### DISCUSSION TITLE
Where do we report bugs with Co-pilot?

### DISCUSSION THREAD

**Discussion by JamesAllerlei** [community/community#114426] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi all, I'm pretty new here. Nooby question but where do we report bugs in co-pilot? I see a pinned discussion with a FAQ and steps for troubleshooting but no general discussion thread or other venue for reporting bugs. Do we just start a general discussion for a problem we are having?

**Comment by celestinod** [comment, marked as answer]
Hey James!

Your question I think is a duplicate of this one: https://github.com/orgs/community/discussions/47872

Suleyman gave a pretty good answer about this topic in the question mentioned above.

Hope you find this helpfull!

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/47872

**Comment by Dika-Amul** [comment]
**saya mengajukan pertanyaan yang mengandung sedikit aritmatika**
"jika seseorang lahir tanggal 6 Juli 1974, maka ulang tahun yang keberapa kah dia tahun ini"

**Dan jawaban yang saya dapatkan dari copilot**
Seseorang yang lahir pada tanggal **6 Juli 1974** akan merayakan **ulang tahun ke-**{{2024-1974}}** tahun** pada tahun ini. 🎂🎉

jawaban Copilot ditak mendapatkan hasil apapun dalam bentuk angka sesuai pertanyaan sebagaimana saya harapkan

sebagai perbandingan saya mengajukan pertanyaan yang sama pada **Chat GPT** dan menghasilkan jawaban yang memuaskan sebagai berikut:
Jika seseorang lahir pada tanggal 6 Juli 1974, maka pada tahun 2024, dia akan merayakan ulang tahunnya yang ke-50.
